owncloudApp/src/main/java/com/owncloud/android/presentation/security/passcode/PassCodeActivity.kt

```diff
@@ -194,7 +194,24 @@ class PassCodeActivity : AppCompatActivity(), NumberKeyboardListener, EnableBiom
     private fun setTextListeners() {
         val numberOfPasscodeDigits = (passCodeViewModel.getPassCode()?.length ?: passCodeViewModel.getNumberOfPassCodeDigits())
         for (i in 0 until numberOfPasscodeDigits) {
-            passCodeEditTexts[i]?.setOnClickListener { hideSoftKeyboard() }
+
+            passCodeEditTexts[i]?.apply {
+                hideSoftKeyboard()
```

Comment:
This doesn't make sense here. Previously this line was inside the `setOnClickListener` method for each of the `EditText`, but we cannot apply it to a `EditText` directly

Reply:
Ok, I notice that when I added the `focus = false` to the `EditTest` the softkeyboard is not shown. So maybe it is better to remove this function because is not useful anymore if we want keep the same behaviour between custom keyboard and external keyboard.